### ISSUE TITLE
*command-line-args* not visible?

### ISSUE THREAD

**Issue by looselytyped** [anmonteiro/lumo#240]
I attempted the following script

```
#!/usr/bin/env lumo

(ns bar.core
  (:require [cljs.core :refer [*command-line-args*]]))

(println (str "Hello " (first *command-line-args*) "!"))
```

And ran it with (after making the script executable) 

```
./test looselytyped
```

The output is 

```
Hello !
```

**Lumo installation details**

```
lumo -v
Lumo 1.7.0
ClojureScript 1.9.908
Node.js v8.4.0
 Docs: (doc function-name-here)
       (find-doc "part-of-name-here")
 Source: (source function-name-here)
 Exit: Control+D or :cljs/quit or exit

cljs.user=> exit
```

**Note** - Apologies if this is a rookie question. I just started playing with Lumo.

**Comment by anmonteiro** [comment]
this has been fixed in #237.

**Comment by looselytyped** [comment]
I apologize @anmonteiro - I looked over the issues and somehow managed to miss that. 

Thank you for the prompt response, and thank you and all the other for all the hard work. 
Really enjoying playing with Lumo so far.
